src/applications/qa/index.js

```diff
@@ -133,10 +135,32 @@ function QuestionAnswering({ application, showSettings }) {
         </div>
         <div className="application__content">
           <h4>What would you like to know?</h4>
-          <div className="qa__input">
+          <Form className="qa__input"
+            onSubmit={evt => {
+              evt.preventDefault();
+
+              // Step 1: Set processing to true
+              setProcessing(true);
+
+              // Step 2: Trigger ask function
+              ask(
+                questionText,
+                retrievers.selectedRetriever,
+                selectedCollection,
+                readers.selectedReader,
+                setAnswers,
+                setProcessing,
+                setProcessed,
+                dispatch
+              );
+
+              searchRef.current.focus();
```

Comment:
Can you add a comment here "Switch focus to question text field"

Reply:
Sure. Comments added.